### PR TITLE
B t17533 adjust print in blueprints using basemap

### PR DESCRIPTION
**Pull request by sierrazamudiodemos**
**Ticket:** https://yaits.demos-deutschland.de/T17533

Description:
1. Update the columns _g_print, default_visibility and enabled to true of _gis table that use basemap and that belong only to masterblueprints.
2. Set _g_print, default_visibility,  enabled to true for the default data whenever a new DemosClient will get demosCore (demosplan/DemosPlanCoreBundle/DataFixtures/ORM/ProdData/LoadProcedureData.php)

### How to review/test
- Create a new Verfahren selecting the "leere Blauepause"
- Once created, go to Planungsdokumente und Planzeichnung > Planzeichenerklärung hochladen 
- Scroll to the bottom to see basemap and the following checkmarks should be checked:
-- Diese Kartenebene wird auch für den Druck von Verortungen in Druckansichten und PDF verwendet
--  Diese Kartenebene beim Laden der Karte anzeigen
--  Kartenebene veröffentlichen 
- Just for information: this tooltip is  that _g_print column that is checked when you click on "bearbeiten"

- [ x] Move the tickets on the board accordingly

### CODE REVIEW

File: demosplan/DemosPlanCoreBundle/DoctrineMigrations/2024/02/Version20240220142305.php [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+    {
+        $this->abortIfNotMysql();
+        // Update gis layers where its procedure is a master template
+        $this->addSql(
+            'UPDATE _gis as g
+        INNER JOIN _procedure AS p ON g._p_id = p._p_id
+        SET g._g_print = 1
+        WHERE g._g_url like :basemapGisUrl
+        AND g._g_name = :basemapGisName
+        AND (p._p_id IN (SELECT _procedure FROM customer) OR p.master_template = 1 OR p._p_master = 1)',
```
Comment (MoritzMandler) [inline demosplan/DemosPlanCoreBundle/DoctrineMigrations/2024/02/Version20240220142305.php:47]: No blocker - just a bit redundant.
SELECT _procedure FROM customer
all of those have to be procedures where
 _p_master is 1.
Customers can only hold a relation to their default-customer-blueprint.

**Review by MoritzMandler — COMMENTED** [review]
Is the core the correct place for this migration or is it a project specific change?

File: demosplan/DemosPlanCoreBundle/DoctrineMigrations/2024/02/Version20240220142305.php (hunk 1 above)
Comment (sierrazamudiodemos) [reply demosplan/DemosPlanCoreBundle/DoctrineMigrations/2024/02/Version20240220142305.php:47]: Hey @MoritzMandler
- it is true. I can remove the customer part as it is redundant.
- yes it should be applicable for all projects

**Review by MoritzMandler — APPROVED** [review]
lgtm

File: demosplan/DemosPlanCoreBundle/DoctrineMigrations/2024/02/Version20240220142305.php (hunk 1 above)
Comment (sierrazamudiodemos) [reply demosplan/DemosPlanCoreBundle/DoctrineMigrations/2024/02/Version20240220142305.php]: @MoritzMandler I updated the sql based on Andreas feedback (I also updated the description in ticket and pr)

**Pull request merged**